project1/application.py

```diff
@@ -31,34 +32,51 @@
 
 @app.route("/")
 def index():
-    return render_template("register.html")
+    return render_template("register.html", flag=True)
 
 @app.route("/register",methods=["GET","POST"])
 def register():
     if (request.method == "POST"):
         username = request.form.get("uname")
         password = request.form.get("pswd")
         dt = datetime.datetime.now()
-        data = Data(Username=username,Password=password,Timestamp=dt)
-        db.session.add(data)
-        db.session.commit()
+        data2 = Data.query.all()
+        for user in data2:
+            if username == user.Username:
+                # return render_template("user.html")
+                return "<h2 Style='color: red;text-align:center'>You already have registered !Please Login </h2>"
```

Comment:
You have to do this step after making sure there are no blank fields. Therefore, this step should be in line #55

Reply:
done